app/views/tags/index.html.erb

```diff
@@ -5,33 +5,35 @@
 </p>
 
 <% @categories.each do |category| %>
-  <h2 id="<%= category.category %>">
-    <%= link_to category.category, category_path(category) %>
+  <div class="tag-title">
+    <h2 id="<%= category.category %>">
+      <%= link_to category.category, category_path(category) %>
+    </h2>
     <% if is_admin %>
-      <span class="byline">| <%= link_to 'edit', edit_category_path(category.category) %></span>
-    <% end %>
-  </h2>
-
-  <% category.tags.each do |tag| %>
-    <ol class="category_tags">
-      <li id="<%= tag.tag %>">
-      <% filtered = @filtered_tags.include?(tag.id) %>
-      <%= tag_link tag %>
-      <span class="<%= filtered || !tag.active ? 'inactive_tag' : '' %>"><%= tag.description %></span>
-      <span class="byline">
-      <% if !tag.active %>
-        | <em>inactive</em>
-      <% end %>
-      <% if filtered %>
-        | <%= link_to "filtered", filters_path(anchor: tag.tag) %>
+      <p class="byline"><span aria-hidden>|</span> <%= link_to 'edit', edit_category_path(category.category), id: "editlink-#{category.category}", aria: {labelledby: "editlink-#{category.category} #{category.category}"} %></p>
       <% end %>
-      <% if is_admin %>
-        | <%= link_to 'edit', edit_tag_path(tag.tag) %>
-      <% end %>
-      </span>
+  </div>
+
+  <ol class="category_tags" aria-labelledby="<%= category.category %>">
+    <% category.tags.each do |tag| %>
+      <li>
+        <% filtered = @filtered_tags.include?(tag.id) %>
+        <%= tag_link tag %>
+        <span class="<%= filtered || !tag.active ? 'inactive_tag' : '' %>"><%= tag.description %></span>
+        <span class="byline">
+          <% if !tag.active %>
+            <span aria-hidden="true">|</span> <em>inactive</em>
```

Comment:
I don't want to add a bunch of work to this PR, but if you wanted to create separate PR with a `divider_tag` helper for this, we could use it in a whole bunch of places across the templates.

Reply:
Great. I used it here. :+1: